acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-deletion-restriction.feature

```diff
@@ -8,24 +8,23 @@ Feature: Piece deletion restrictions from order and order line
     * def okapitokenUser = okapitoken
     * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
     * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*' }
-    * configure headers = headersUser
 
     * callonce variables
     * def last_piece_error_masage = "The piece cannot be deleted because it is the last piece for the poLine in with Receiving Workflow 'Synchronized order and receipt quantity' and cost quantity '1'"
+    * def fundId = call uuid
+    * def budgetId = call uuid
 
+    # Prepare finances
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)' }
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 10000, 'fundId': '#(fundId)' }
+    * configure headers = headersUser
 
   Scenario: Avoid deltion of piece when order has status 'Closed' and poLine cost quantity '1' and ReceivingWorkflow 'Syncronized'
```

Comment:
A small typo: "Avoid deletion..." & can add @Positive/@Negative indicators as usual

Reply:
Thanks, fixed

Reply:
As for Annotation, both scenarios will check for sucess and exception case, this is why didn't add